documentation/asciidoc/computers/configuration/host-wireless-network.adoc

```diff
@@ -0,0 +1,165 @@
+== Host a wireless network on your Raspberry Pi
+
+Your Raspberry Pi can host its own wireless network using a wireless module.
+If you connect your Raspberry Pi to the internet via the Ethernet port (or a second wireless module), other devices connected to the wireless network can access the internet through your Raspberry Pi.
+
+Consider a wired network that uses the `10.x.x.x` IP block. You can connect your Raspberry Pi to that network and serve wireless clients on a separate network that uses another IP block, such as `192.168.x.x`.
+
+////
+In the diagram below, note that the laptop exists in an IP block separate from the router and wired clients:
+
+----
+                                         +- RPi -------+
+                                     +---+ 10.10.0.2   |          +- Laptop ----+
+                                     |   |     WLAN AP +-)))  (((-+ WLAN Client |
+                                     |   | 192.168.1.1 |          | 192.168.1.2 |
+                                     |   +-------------+          +-------------+
+                 +- Router ----+     |
+                 | Firewall    |     |   +- PC#2 ------+
+(Internet)---WAN-+ DHCP server +-LAN-+---+ 10.10.0.3   |
+                 |   10.10.0.1 |     |   +-------------+
+                 +-------------+     |
+                                     |   +- PC#1 ------+
+                                     +---+ 10.10.0.4   |
+                                         +-------------+
+----
+TODO: Switch to an image version of this ascii art!
+////
+
+With this network configuration, wireless clients can all communicate with each other through the Raspberry Pi router. However, clients on the wireless network cannot directly interact with clients on the wired network other than the Raspberry Pi; wireless clients exist in a private network separate from the network that serves wired clients.
+
+NOTE: The Raspberry Pi 4, Raspberry Pi 3, and Raspberry Pi Zero W can host a wireless network using the built-in wireless module. Raspberry Pi models that lack a built-in module support this functionality using a separate wireless dongle.
+
+=== Prerequisites
+
+To host a wireless network with your Raspberry Pi, you will need the following:
+
+* a Raspberry Pi with a wireless module (built-in or separate)
+* administrative access to your Raspberry Pi
```

Comment:
I think throughout the documentation we **always** assume that the reader has "administrative access" to their Raspberry Pi! (so it feels weird seeing it mentioned here)

Reply:
Fair point. @nathan-contino the default account on a Pi is always in the sudo'ers file. Can you do a quick edit and remove these references.